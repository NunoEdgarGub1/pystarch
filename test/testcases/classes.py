

class Test:
    def __init__(self):
        self.x = 1
        self.y = '2'

    def f(self, a):
        return a + self.x


t = Test()
a = t.x
b = t.y
c = t.f(2)
