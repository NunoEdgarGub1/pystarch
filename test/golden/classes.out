Test Test None
a Num 1
b Str 2
c Num
t Instance(Test)
